packages/angular_devkit/build_angular/src/webpack/utils/stats.ts

```diff
@@ -124,7 +147,20 @@ function statsToString(json: any, statsConfig: any, bundleState?: BundleStats[])
     unchangedChunkNumber = json.chunks.length - changedChunksStats.length;
   }
 
-  const statsTable = generateBuildStatsTable(changedChunksStats, colors);
+  // Sort chunks by size in descending order
+  const changedChunksStatsDesc = changedChunksStats.sort((a, b) => {
+    if (a.stats[2] > b.stats[2]) {
+      return -1;
+    }
+
+    if (a.stats[2] < b.stats[2]) {
+      return 1;
+    }
+
+    return 0;
+  });
```

Comment:
nit: this could be a simple subtraction:

```suggestion
  const changedChunksStatsDesc = changedChunksStats.sort((a, b) => b.stats[2] - a.stats[2]);
```

Also, `sort` operates in-place so does in fact modify `changedChunksStats`, which makes `changedChunksStatsDesc` identical to `changedChunksStats`. I'd remove `changedChunksStatsDesc` to avoid confusion, or explicitly copy `changedChunksStats` before sorting.

Reply:
Good catch on the mutation operation.

The size here can also be a string which will be `NaN`, which actually makes the above the simplest IMHO.

Reply:
Ah, I did not consider the `NaN` case, fair enough :-)